src/mds/topology/topology_metric.cpp

```diff
@@ -232,6 +234,29 @@ void TopologyMetricService::UpdateTopologyMetrics() {
         }
     }
 
+    // cluster io
+    uint64_t clusterReadIOPS = 0, clusterWriteIOPS = 0,
```

Comment:
It isbettrer to keep the data type consistent with metric

Reply:
fix